.github/workflows/pest-tests.yml

```
@@ -0,0 +1,91 @@
+    - name: Run Database Migrations
+      run: php artisan migrate --force
+
+    - name: Run Pest Tests
+      run: php artisan test --parallel --coverage-clover=coverage.xml
+
+    - name: Upload Coverage to Codecov
+      if: matrix.php-version == '8.4'
+      uses: codecov/codecov-action@v4
+      with:
+        file: ./coverage.xml
+        fail_ci_if_error: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)